Sources/Overlays/_Testing_AppKit/Attachments/NSImage+AttachableAsCGImage.swift

```diff
@@ -0,0 +1,95 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+#if SWT_TARGET_OS_APPLE && canImport(AppKit)
+public import AppKit
+@_spi(ForSwiftTestingOnly) @_spi(Experimental) public import _Testing_CoreGraphics
```

Comment:
`@_spi(ForSwiftTestingOnly)` has been phased out since this PR was originally posted, and we can use `package` now.